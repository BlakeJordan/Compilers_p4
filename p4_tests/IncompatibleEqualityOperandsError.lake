bool g() {
    return true;
}

bool f() {
    int a;
    a = 5;
    return a == g();
}