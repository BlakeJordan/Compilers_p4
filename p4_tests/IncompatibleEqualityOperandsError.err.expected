8,14: Invalid equality operation
Type checking failed
